tests/e2e/conftest.py

```diff
@@ -63,16 +63,16 @@
 PLATFORM_SOURCE_PREFIXES = {
     # neuro mkdir -p storage:e2e/assets/data
     # neuro cp -rT tests/assets/data storage:e2e/assets/data
-    "storage": "storage:e2e/assets/data",
+    # "storage": "storage:e2e/assets/data",
```

Comment:
Currently copying to/from platform storage is broken due to https://github.com/neuro-inc/platform-api/issues/2088